android/src/main/java/com/tanguyantoine/react/MusicControl.java

```diff
@@ -19,6 +19,10 @@
         return modules;
     }
 
+    public List<Class<? extends JavaScriptModule>> createJSModules() {
+        return Collections.emptyList();
+    }
+
```

Comment:
This has been removed from RN ages ago. As you're now retrieving it from the jcenter repository instead of npm, you're getting an older version.

Reply:
Oh, sorry! Will revert!